docs/guides/building-a-minecraft-demo.md

```diff
@@ -382,26 +382,28 @@ teleport to the end of the arc. Before, we wrote our own A-Frame components.
 But we can also use open source components already made from the community
 and just use them straight from HTML!
 
-[blink-controls]: https://github.com/fernandojsg/aframe-teleport-controls/
-For teleportation, there's a component called [blink-controls][blink-controls].
-Following the README, we add the component via a `<script>` tag and just set
-the `blink-controls` component on the controller on the entity:
+Even though we haven't explicity defined one, our scene has an implicit camera 
```

Comment:
Is this necessary? Could just say `we define a player entity that contains the controller and the camera`? Not sure talking about default / implicit camera adds much value here.

Reply:
I wanted to provide an answer to "Why do we suddenly need to define a player entity now?" but I think you're right.